src/VirtualClient/VirtualClient.Packaging/ctstraffic.vcpkg

```diff
@@ -0,0 +1,6 @@
+﻿{
```

Comment:
You have weird char here

Reply:
Can u still see it? I dont.

Reply:
<img width="706" alt="image" src="https://github.com/microsoft/VirtualClient/assets/31965446/8640e3bc-6605-4015-b91a-f30a645192cf">

Reply:
It is gone now